stage1/usr_from_coreos/manifest-amd64-usr.d/journald.manifest

```diff
@@ -1,5 +1,7 @@
+lib64/tmpfiles.d/journal-nocow.conf
```

Comment:
Do we need this? And if we do, can we add it for the host flavor and ARM too?

The docs in the file say it only matters for btrfs filesystems and it improves performance.

Reply:
Oops, that's a left-over from an experiment. I'll remove it from this PR.